### DISCUSSION TITLE
Banner image

### DISCUSSION THREAD

**Discussion by aubymori** [ALTaleX531/OpenGlass#11] [Show and tell]
I noticed that the README simply re-uses the banner from the glass8.eu website, so I took it upon myself to make my own image to show off OpenGlass.

![openglass](https://github.com/ALTaleX531/OpenGlass/assets/44238627/f507e3e8-a4bd-4edd-8386-e221650c6ca9)

**Wallpaper:** [metalheart jawn #2](https://www.deviantart.com/kfh83/art/metalheart-jawn-2-1068250045) by @kfh83

**Comment by ALTaleX531** [comment]
Looks great, I'll update the README and provide credits later.
